lib/main.coffee

```diff
@@ -154,14 +154,14 @@ module.exports =
               range: [[lineIdx, 0], [lineIdx, colEndIdx]]
             ]
           else
-            atom.notifications.addError """
-              **sass-lint had a problem**
-              Please consider filing an issue with [linter-sass-lint](https://github.com/AtomLinter/linter-sass-lint)
-              or [sass-lint](https://github.com/sasstools/sass-lint) including the text below and any other
-              information possible.
-
-              #{error.stack}
-            """, {dismissable: true}
+            # Leaving this here to allow people to report the errors
+            console.log(error.stack)
```

Comment:
Yeah good show will do, just hotfixing this for the moment as it's bugging me as i use it! Will handle these errors a lot better in 1.1.0